spec/support/spec_config.rb

```diff
@@ -92,6 +92,23 @@ def connect_options
     end
   end
 
+  def supports_scram_256?
```

Comment:
This method should be in ClusterConfig which deals with how the cluster is configured rather than SpecConfig, which deals with how the test suite is configured.